boards/arm/mimxrt1064_evk/mimxrt1064_evk.dts

```diff
@@ -160,6 +162,35 @@ arduino_i2c: &lpi2c1 {};
 	};
 };
 
+&flexspi2 {
+	status = "okay";
+};
+
+&w25q32jvwj0 {
```

Comment:
Now we can remove mimxrt1064_evk from platform_exclude in samples/subsys/usb/dfu/sample.yaml. Would be nice to have partitions for other mimxrts.